launch/tier4_perception_launch/launch/object_recognition/tracking/tracking.launch.xml

```diff
@@ -69,20 +69,24 @@
   <group if="$(var use_multi_channel_tracker_merger)">
     <let name="detection_enabled_channels" value="$(var input/detection_input_channels)"/>
     <group scoped="false" if="$(eval '&quot;$(var mode)&quot;==&quot;camera_lidar_radar_fusion&quot;')">
-      <let name="detection_enabled_channels" value="['lidar_$(var lidar_detection_model)_validated','camera_lidar_fusion','detection_by_tracker','radar_far']" if="$(var use_detection_by_tracker)"/>
-      <let name="detection_enabled_channels" value="['lidar_$(var lidar_detection_model)_validated','camera_lidar_fusion','radar_far']" unless="$(var use_detection_by_tracker)"/>
+      <let
+        name="detection_enabled_channels"
+        value="['lidar_$(var lidar_detection_model_type)_validated','camera_lidar_fusion','detection_by_tracker','radar_far']"
+        if="$(var use_detection_by_tracker)"
+      />
+      <let name="detection_enabled_channels" value="['lidar_$(var lidar_detection_model_type)_validated','camera_lidar_fusion','radar_far']" unless="$(var use_detection_by_tracker)"/>
```

Comment:
LiDAR detection validation is optional. See `use_validator` parameter used in detection.launch.xml.
So, I think we need to add `'lidar_$(var lidar_detection_model_type)'` in every part of this launch.

Reply:
thank you for the comment. it is fixed https://github.com/autowarefoundation/autoware.universe/pull/9624/commits/7dc931bc50d8a6cda3ea73b7346f08fb3a8afdbf